lib/orderbook/OrderBook.ts

```diff
@@ -100,6 +100,16 @@ class OrderBook extends EventEmitter {
     // TODO: implement
   }
 
+  public removeOrder = (orderId: string, pairId: string): boolean => {
+    const matchingEngine = this.matchingEngines[pairId];
+    if (!matchingEngine) {
+      throw errors.INVALID_PAIR_ID(pairId);
+    }
+
+    return matchingEngine.removeOrder(orderId);
+    // TODO: remove from this.ownOrders/this.peerOrders as well
```

Comment:
Why not implement this as part of this PR? Should be easy enough, to do something like `ownOrders[orderId] = undefined` right?

Reply:
Oh I didn't see your explanation @moshababo in your comment above. I still think it makes sense to put it in here because it can be done in one line (correct me if I'm wrong) but I'll leave it up to you.

Reply:
Yes it make sense, I implemented `removeOrder` function on `OrderBook`. The naming is a bit confusing, but we'll live with that until we do the refactoring.